### ISSUE TITLE
How to connect the model to an exoskeleton?

### ISSUE THREAD

**Issue by MohammadSahandi** [lsw9021/MASS#44]
Hi, 
I want to know the method for using this environment for exoskeleton training with DRL. First how can I import exoskeleton to the environment and then how to connect it to human model?
Thanks

**Comment by emanay1414** [comment]
Following

**Comment by todayThursday** [comment]
yes，I also have same question

**Comment by todayThursday** [comment]
Yes, I'm also facing this problem now. May I ask if you have solved it? Perhaps I can get some inspiration from you. 

> Hi, I want to know the method for using this environment for exoskeleton training with DRL. First how can I import exoskeleton to the environment and then how to connect it to human model? Thanks

**Comment by mxhjs29** [comment]
@todayThursday Hi, have you solved this problem? I've solved this in mujoco, but cannot do this in dart.
